frontend/Makefile

```diff
@@ -40,6 +40,9 @@ release-build:
 ##################################################
 # Local development
 ##################################################
+build: # Build the Next.js local dev server in Docker
+	docker-compose build --no-cache nextjs
+
```

Comment:
There might be a better option for updating local packages in docker - but this was the quick solution